.github/workflows/ci.yml

```diff
@@ -159,6 +159,30 @@ jobs:
         run: yarn test
         env:
           POSTGRES_EXTERNAL_URL: postgresql://admin:admin@localhost:5433/labelflow?schema=public
+      - name: upload coverage artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: jest code coverage
+          path: coverage/lcov.info
+
+  publish-tests-coverage:
+    name: Publish tests code coverage to Code Climate
+    runs-on: ubuntu-20.04
+    needs: test-typescript
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: download code coverage artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: jest code coverage
+      - name: publish jest code coverage to codeclimate
+        uses: paambaati/codeclimate-action@v3.0.0
+        env:
+          CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
+        with:
+          coverageLocations: |
+            lcov.info:lcov
```

Comment:
What is this for?

Reply:
for information the `upload-artifact` doesn't keep the full path of `coverage/lcov.info`, when I do `download-artifact` I get a `lcov.info` put at the root folder.

Reply:
Yes, that's normal since you're using an artifact instead of the cache 😉